### ISSUE TITLE
Scoreboard Flickers

### ISSUE THREAD

**Issue by Swedz** [TigerHix/ScoreboardLib#4]
The scoreboard flickers. I have a gif of it here: https://i.gyazo.com/c48b2259d3bd6bbe431eb6f75d3ff1e7.mp4

**Here is my code:**

```
public class ScoreboardManager implements ScoreboardHandler {
    public List<Entry> getEntries(Player player) {
        EntryBuilder eb = new EntryBuilder();

        eb.next("§8§l§m================");
        eb.next("§2§lFaction: §fMemes");
        eb.next(" §7► §aPlayers: §f1/1");
        eb.next(" §7► §aDTR: §f1/1");
        eb.next("§8§l§m================");
       
        return eb.build();
    }

    public String getTitle(Player player) {
        return "§e§lMCCubed§r §7(Map 1)§r";
    }
}
```

Some info that could be needed:

- Running Spigot-1.7.9-R0.2-208-ge0f2e95 (MC: 1.7.10) (Implementing API version 1.7.10-R0.1-SNAPSHOT)
- Code is just what is shown, I use some mechanics to get the values.
- I have tried removing mechanics to see if it was somehow slowing it down to cause an issue.
- It seems to only occur on the first line, and that it only removes a portion of the text. Is this possibly something to do with the slider texts even though I'm not using one?

**Comment by TigerHix** [comment]
This is a duplicate of #3.

I didn't come up with a good internal workaround, but try replacing the second separator line, from

```
eb.next("§8§l§m================");
```

to:

```
eb.next("§l§m§8================");
```

**Comment by TigerHix** [comment]
If it helped, please do me a favor and star the project ^^

**Comment by Swedz** [comment]
It's still occurring. I have also just tried it without the separator lines and the "Faction: Memes" line does it when it's the first line.

It is definitely something with the first line.

Edit: I find that it's consistently the first line and sometimes it occurs on the other lines.

**Comment by Garcia-Florian** [comment]
I have the same issue

**Comment by TigerHix** [comment]
@Swedz Really weird. Try using the demo code, does the first line flicker too?

By the way, sometimes if you see very random, short interval flickers, that's mostly because network lag. Players with high ping have reported that they would see the scoreboard flickering while players with low ping don't.

**Comment by Swedz** [comment]
I get like 30 ping or less to my server and I see the flickering. And it's not really a "flicker" it's more that the first line gets messed up in some sort of substring() way.

Where could I find the demo code?

**Comment by TigerHix** [comment]
Yes, I was referring to the actual "flickering" (the lines disappear
quickly and show up again). This issue from yours (text on a line gets
cleared and reappear) is quite different.

By demo code I refer to the usage example in README.md:
https://github.com/TigerHix/ScoreboardLib/blob/master/README.md

**Comment by Swedz** [comment]
Okay, I'll be sure to try this out when I get home, as I am not home at the moment. I'll reply with the results of the test.

**Comment by Garcia-Florian** [comment]
Look, I have the same issue and I made a video to let you see the flickering
I also use the demo code 😞 
https://www.youtube.com/watch?v=numZ6S5iuS0

**Comment by TigerHix** [comment]
Indeed very weird. My server has been using 1.9.4 for a long time and I haven't updated to latest versions. Are you guys using 1.11?

**Comment by Swedz** [comment]
I said in my original post I am using

```
Running Spigot-1.7.9-R0.2-208-ge0f2e95 (MC: 1.7.10) (Implementing API version 1.7.10-R0.1-SNAPSHOT)
```

**Comment by tomocrafter** [comment]
I have same problem, Even if I do the same thing as README.md, flickering will occur.
